docs/dev-guides/4-technical-notes.md

```diff
@@ -0,0 +1,16 @@
+---
+id: technical-notes
+title: Technical Notes
+---
+
+This page contains technical notes that might help in development of specific kinds of XRPKs.
+
+## Avatars
+
+Avatars are applications that you can control and view in first-person mode to see their perspective. They can be anything from a sword to a person, packaged into an XRPackage.
```

Comment:
I think this might be confusing; avatars aren't really applications. They are just a representation of the user in the world.

This is clarified in later paragraphs but I think it starts the user off on the wrong foot in their understanding.

Reply:
Thanks, updated!